README.md

```diff
@@ -62,43 +76,35 @@ The Terminal-based user interface provides a simple way to watch the status of e
     ```
    - [ ] Then, proceed to install, play, and open the TUI
 
-    __Note__ Error logs are available for each task:
+ __Note__ Error logs are available for each task:
     ```
     cat ~/cylc-run/<workflow-name>/<run#>/log/job/1/<task-name>/01/job.err
     ```
+   
+### Running the Cylc pipeline locally
```

Comment:
Does this work on Windows? I was trying the old instructions and I couldn't get things to run.

Reply:
That seems like another can of worms to address in a different PR. I don't see any specific instructions for installing Cylc on Windows. We'd have to log errors and figure out what the issue is. I'll start a new issue for it.

Reply:
Using the docker container is the goal for pipeline work to avoid the issue of different builds on different architectures :). But for local development, users will have to build the project as outlined in the new section at the bottom of the readme.